package.json

```diff
@@ -279,5 +280,6 @@
   },
   "files": [
     "dist"
-  ]
+  ],
+  "type":"module"
```

Comment:
Is this change necessary?

Reply:
no, revert this

Reply:
I had to do these to be able to use ```import``` keyword.

Reply:
nope, there is a seperate `package.json` in tools directory, which already do this, works fine for the other tools

Reply:
no, please rename file extension to `mjs` and it should work as is.

Reply:
Sure thanks..I for some reason thought I should keep it ```js```